web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -34,7 +34,8 @@
 	$: currentAsset = currentMemory?.assets[assetIndex];
 	$: nextAsset = currentMemory?.assets[assetIndex + 1];
 
-	$: canAdvance = !!(nextMemory || nextAsset);
+	$: canGoForward = !!(nextMemory || nextAsset);
+	$: canGoBack = !!(previousMemory || previousAsset);
```

Comment:
for consistency, `Backward` might be better.

Reply:
Though, semantically `Back` does make more sense...

Reply:
`hasPrevious` and `hasNext`?